### PR TITLE
Fixed #24677 -- made TextField to_python return a string

### PR DESCRIPTION
**Pull request by rolo**
Small fix for #24677. There was no TextField specific test class to went ahead and added one.

### CODE REVIEW

File: tests/model_fields/tests.py [hunk 1]
```diff
@@ -246,6 +246,13 @@ class Meta:
         )
 
 
+class TextFieldTests(test.TestCase):
+    def test_to_python(self):
+        """TextField.to_python should return a string"""
+        f = models.CharField()
```
Comment (bronsen) [inline tests/model_fields/tests.py]: do you mean **models.TextField()** here?

**Comment by rolo** [comment]
Doh! Apologies. Have corrected.

**Comment by timgraham** [comment]
buildbot, add to whitelist.

**Comment by timgraham** [comment]
merged in 19e67c6cd13d6442abb2a6eba1b7fba80dcf5457, thanks!

**Pull request closed without merging**